cmd/cluster-agent/api/v1/clusterchecks.go

```diff
@@ -19,12 +19,10 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/clusteragent/clusterchecks"
 	cctypes "github.com/DataDog/datadog-agent/pkg/clusteragent/clusterchecks/types"
 	"github.com/DataDog/datadog-agent/pkg/config"
+	dcautil "github.com/DataDog/datadog-agent/pkg/util/clusteragent"
```

Comment:
is the `dcautil` alias mandatory here ?

Reply:
i think to distinguish from `"github.com/DataDog/datadog-agent/pkg/clusteragent"` ?

Reply:
indeed 👍